src/legacy/core_plugins/metrics/public/request_handler.js

```diff
@@ -17,13 +17,13 @@
  * under the License.
  */
 
-import { validateInterval } from '../lib/validate_interval';
+import { validateInterval } from './lib/validate_interval';
 import { timezoneProvider } from 'ui/vis/lib/timezone';
 import { timefilter } from 'ui/timefilter';
 import { kfetch } from 'ui/kfetch';
 
-const MetricsRequestHandlerProvider = function(Private, config) {
-  const timezone = Private(timezoneProvider)();
+const MetricsRequestHandlerProvider = function(config) {
+  const timezone = timezoneProvider();
```

Comment:
timezoneProvider requires passing config in

Reply:
also it returns a function, which you should call to get the timezone

`const timezone = timezoneProvider(config)();`

Reply:
Done